### PR TITLE
vim-patch:c51cf0329809

### PR DESCRIPTION
**Pull request by clason**
Update runtime files.
https://github.com/vim/vim/commit/c51cf0329809c7ae946c59d6f56699227efc9d1b

I think I got all the applicable "`.` -> `..`" changes?

### CODE REVIEW

File: runtime/doc/tips.txt [hunk 1]
```diff
@@ -286,12 +286,12 @@ line above and below each list entry.  The expression commands are used to
...
-	:let m = m . "i     <CR>     <Esc>"    " add space for indent
-	:let m = m . "gq}"		      " format text after the bullet
-	:let m = m . "{dd"		      " remove the empty line
-	:let m = m . "5lDJ"		      " put text after bullet
+	:let m = m .. "{O<Esc>"		      " add empty line above item
+	:let m = m .. "}{)^W"		      " move to text after bullet
+	:let m = m .. "i     <CR>     <Esc>"    " add space for indent
+	:let m = m .. "gq}"		      " format text after the bullet
+	:let m = m .. "{dd"		      " remove the empty line
+	:let m = m .. "5lDJ"		      " put text after bullet
```
Comment (seandewar) [inline runtime/doc/tips.txt:294]: patch used `..=`

https://github.com/vim/vim/commit/c51cf0329809c7ae946c59d6f56699227efc9d1b#diff-910c5937c45fb1791c4e25b9883bffed6623eca6ea6fe9e25773066a322ecf4cR359-R365

**Review by seandewar — COMMENTED** [review]
missing `builtin.txt` changes for:

- `getchar`
- `map`
- `mkdir`
- `readdir`
- `remote_peek`
- `remote_send`
- `screencol`
- `searchcount`
- `shellescape`
- `substitute`
- `tempname`
- `trim`

others:
- https://github.com/vim/vim/commit/c51cf0329809c7ae946c59d6f56699227efc9d1b#diff-b3aeef96275167ddcc1e5fee6d51b8f2018f9359f921d86ad6e7035ca80f20e9R505
- https://github.com/vim/vim/commit/c51cf0329809c7ae946c59d6f56699227efc9d1b#diff-b3aeef96275167ddcc1e5fee6d51b8f2018f9359f921d86ad6e7035ca80f20e9R3150
- https://github.com/vim/vim/commit/c51cf0329809c7ae946c59d6f56699227efc9d1b#diff-da34d85d1945e9394688b2473eea604d6529c841af0dd38001d2fa2e421e975aR930
- https://github.com/vim/vim/commit/c51cf0329809c7ae946c59d6f56699227efc9d1b#diff-442131dbc53d601ff38edcc95c73c09272934f92f7cf6601738cc6ceaf8a1e48R1170
- https://github.com/vim/vim/commit/c51cf0329809c7ae946c59d6f56699227efc9d1b#diff-442131dbc53d601ff38edcc95c73c09272934f92f7cf6601738cc6ceaf8a1e48R4890
- https://github.com/vim/vim/commit/c51cf0329809c7ae946c59d6f56699227efc9d1b#diff-9df508bed04b2c775c4a5a084ad489250d335a118a6cc3505a915d2348fcde3cR987

**Comment by clason** [comment]
Most of these I already did? (Sorry for the wrong fixup message; it'll get squashed anyway.)

File: runtime/doc/tips.txt [hunk 2]
```diff
@@ -286,12 +286,12 @@ line above and below each list entry.  The expression commands are used to
...
 	:let m =     ":map _f  :set ai<CR>"    " need 'autoindent' set
-	:let m = m . "{O<Esc>"		      " add empty line above item
-	:let m = m . "}{)^W"		      " move to text after bullet
-	:let m = m . "i     <CR>     <Esc>"    " add space for indent
-	:let m = m . "gq}"		      " format text after the bullet
-	:let m = m . "{dd"		      " remove the empty line
-	:let m = m . "5lDJ"		      " put text after bullet
+	:let m ..= "{O<Esc>"		      " add empty line above item
+	:let m ..= "}{)^W"		      " move to text after bullet
+	:let m ..= "i     <CR>     <Esc>"    " add space for indent
```
Comment (seandewar) [inline runtime/doc/tips.txt:291]: patch made the comments line up too

**Review by seandewar — COMMENTED** [review]
> Most of these I already did

hmm, i still can't see the missing builtin.txt changes :thinking:
(the others have been addressed tho)

File: runtime/doc/tips.txt (hunk 2 above)
Comment (clason) [reply runtime/doc/tips.txt:291]: ooh, fancy patch!

**Comment by seandewar** [comment]
now i see them! :rocket:

**Review by seandewar — COMMENTED** [review]
Nearly there!

- `remote_peek`
- `remote_send`
- `screencol`
- `searchcount`

File: runtime/doc/builtin.txt [hunk 3]
```diff
@@ -5950,12 +5950,12 @@ remote_send({server}, {string} [, {idvar}])
 		Note: Any errors will be reported in the server and may mess
 		up the display.
 		Examples: >
-		:echo remote_send("gvim", ":DropAndReply ".file, "serverid").
+		:echo remote_send("gvim", ":DropAndReply " .. file, "serverid").
```
[the remark this answers is not in the document]
Comment (clason) [reply runtime/doc/builtin.txt]: Man, they really made sure I you have to use _all of the_ regexes to catch these changes...

**Review by seandewar — APPROVED** [review]
freedom! :rocket:

**Pull request merged**